### ISSUE TITLE
ROT!

### ISSUE THREAD

**Issue by v-oxel** [extremecodetv/ExtremeCodeOS#25]
ROT чисто физически необходим системе, так как это действительно полезно. На данный момент он работает лишь в пососной консольке, но я уверен найдется специалист сделающий GUI. А вот ссыль на ROT https://github.com/v-oxel/rot/.

**Comment by extremecodetv** [comment]
Да, по поводу безопасности и сохранности данных сейчас большой хайп.
Нужно обязательно добавить, мы тогда будем самой безопасной ОСью нового поколения
